server/routes/tenants.js

```diff
@@ -33,4 +52,34 @@ router.post("/properties/:id/tenant", async(req, res) => {
     }
 })
 
-module.exports = router;
\ No newline at end of file
+// update Tenant
+router.put('/', async(req, res, next) => {
+    const { id, firstName, lastName, email, lease, propertyId } = req.body;
+    try {
+    const foundTenant = await Tenant.updateOne({tenantid: id}, {$set: 
```

Comment:
Same point as above: 

 (I suggest testing this out using the dev script) 

```suggestion
    const foundTenant = await Tenant.updateOne({_id: id}, {$set: 
```